include/seqan3/alphabet/composite/alphabet_variant.hpp

```diff
@@ -214,24 +214,24 @@ class union_composition : public alphabet_base<union_composition<alternative_typ
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    constexpr union_composition() noexcept : base_t{} {}                           //!< Defaulted.
-    constexpr union_composition(union_composition const &) = default;              //!< Defaulted.
-    constexpr union_composition(union_composition &&) = default;                   //!< Defaulted.
-    constexpr union_composition & operator=(union_composition const &) = default;  //!< Defaulted.
-    constexpr union_composition & operator=(union_composition &&) = default;       //!< Defaulted.
-    ~union_composition() = default;                                                //!< Defaulted.
+    constexpr alphabet_variant() noexcept : base_t{} {}                           //!< Defaulted.
```

Comment:
indentation :nail_care: :nail_care: :nail_care: 
also why is only the default ctor noexcept?

Reply:
> indentation nail_care nail_care nail_care

not my fault, is it?

> also why is only the default ctor noexcept?

Because originally it wasn't constexpr and the other were (which implied noexcept in cpp17)....